docs/UserGuide.md

```diff
@@ -453,7 +512,14 @@ Format: `exit`
 All tCheck data (i.e. contact details, ingredient data, sales data) are saved in the hard disk automatically after any
  command that changes the data. There is no need to save manually by entering any command.
  
- 
+## 5. FAQ <a name="FAQ"></a>
+
+**Q**: Why for single-word commands like `help`, it will ignore extra inputs that I inputted? <br>
+**A**: Any extra inputs after the single-word command, for example, the input `123` inside the input `c-list 123`, will
+ be ignored because extra inputs does not affect the operation conducted by `c-list`. These single-word commands
+ include: `s-list`, `s-rank`, `i-set-default`, `i-list`, `i-reset-all`, `i-restock`, `c-list`, `c-today`, `c-tomorrow`,
+  `c-clear`, `c-archive-all`, `c-archive-list`, `help`, `exit`.
+
```

Comment:
Could be better phrased as such:
"Q: Why does tCheck ignore additional/ extra input that I add after single-word commands, like `help`"?

Additionally,
* I think it should just be "input" instead of "inputs" since the entire line entered is the input?
* it should be "a" instead of "the" in "after the single-word command"
* if "input" is used, then "extra input does not affect ..." OR if "inputs" is used, then "extra inputs do not affect ..."

Reply:
Okay thanks!